splice_access_api/src/main/java/com/splicemachine/access/configuration/SQLConfiguration.java

```diff
@@ -200,6 +200,9 @@
     public static final String PARTITIONSERVER_JMX_PORT = "hbase.regionserver.jmx.port";
     private static final int DEFAULT_PARTITIONSERVER_JMX_PORT = 10102;
 
+    public static final String NESTEDLOOPJOIN_BATCH_SIZE = "splice.nestedLoopJoin.batchSize";
+    private static final int DEFAULT_NESTEDLOOPJOIN_BATCH_SIZE = 10;
```

Comment:
we didn't want to make this a URL hint, so that we could modify it on a per-query basis?